webapp/graphite/render/functions.py

```diff
@@ -2958,6 +2957,33 @@ def reduceSeries(requestContext, seriesLists, reduceFunction, reduceNode, *reduc
     metaSeries[key].name = key
   return [ metaSeries[key] for key in keys ]
 
+def applyByNode(requestContext, seriesList, nodeNum, templateFunction, newName=None):
+  """
+  Takes a seriesList and applies some complicated function (described by a string), replacing templates with uniquified
```

Comment:
I don't think that "uniquified" is a real word, perhaps "normalized" has the same connotation you're looking for?